addons/website/static/src/components/fields/fields.js

```diff
@@ -32,7 +32,17 @@ class PageUrlField extends UrlField {
             (inputEl) => {
                 if (inputEl) {
                     const fireChangeEvent = () => {
-                        inputEl.dispatchEvent(new Event("change"));
+                        // handle out-of-date view without show_redirect_old_url
+                        if (!this.props.record.fields.show_redirect_old_url) {
+                            inputEl.dispatchEvent(new Event("change"));
+                            inputEl.removeEventListener("input", fireChangeEvent);
+                            return;
+                        }
```

Comment:
In this "out-of-date" case (which will be thousands of people), since you remove the event handler, does it mean that you can never re-hide the "redirect" section if you come back to the original URL? Actually, I just tested quickly in master without your fix, it seems already buggy before, maybe there is better to do?

Reply:
technically yes (the first input + each blur with the normal behavior), but it already never re-hide the div currently.

I've not understood why yet, my guess was that this was some form view shenanigan (that don't match every feature from normal owl view).

huvw suggested from another fix he did, that we should remove the onchange and use another field that doesn't trigger onchange, and when doing that change by adding the field, the invisible attribute is recomputed.